src/README.md

```diff
@@ -0,0 +1,25 @@
+# TTADP Backend
+
+The backend is built using these primary tools:
+
+Production:
+- Express v4 (Web Framework)
+- Sequelize v5 (Database ORM)
+
+Development:
+- Jest (Testing)
+
+## Required Reading
+
+If you are not familiar or need a refresher, please read up on the following resources:
+
+### Express
+- [Official Documentation](https://expressjs.com/)
+
+
+### Sequelize
+- [Official Documentation](https://sequelize.org/v5/)
+
+### Jest
+- [Offical Documentation](https://jestjs.io/docs/en/getting-started)
+- [Best Practives](https://github.com/goldbergyoni/javascript-testing-best-practices/)
```

Comment:
Please let me know if you have any resources I can add

Reply:
I've found the react testing library cheat sheet helpful, https://testing-library.com/docs/react-testing-library/cheatsheet/